Supply module spec (#4599)
cosmos/cosmos-sdk: docs/spec/governance/01_concepts.md

module's proposal handler when a proposal passes. This custom handler may perform
arbitrary state changes.

## Deposit

> To prevent spam, proposals must be submitted with a deposit in a the coins defined in the `MinDeposit` param. The voting period will not start as long as the proposal's deposit is smaller than the `MinDeposit`.

Reviewer: ```suggestion
To prevent spam, proposals must be submitted with a deposit in the coins defined in the `MinDeposit` param. The voting period will not start until the proposal's deposit equals `MinDeposit`.
```

> When a proposal is submitted, it has to be accompanied by a deposit that must be strictly positive, but can be inferior to `MinDeposit`. The submitter doesn't need not pay for the entire deposit on their own. If a proposal's deposit is inferior to `MinDeposit`, other token holders can increase the proposal's deposit by sending a `Deposit` transaction. The deposit is kept in an escrow in the governance `ModuleAccount` until the proposal is finalized (passed or rejected).

Reviewer: ```suggestion
When a proposal is submitted, it has to be accompanied by a deposit that must be strictly positive, but can be inferior to `MinDeposit`. The submitter doesn't need to pay for the entire deposit on their own. If a proposal's deposit is inferior to `MinDeposit`, other token holders can increase the proposal's deposit by sending a `Deposit` transaction. The deposit is kept in an escrow in the governance `ModuleAccount` until the proposal is finalized (passed or rejected).
```
